src/cli/run.ts

```diff
@@ -75,14 +75,7 @@ export const run = async (init: d.CliInitOptions) => {
     loadedCompilerLog(sys, logger, flags, coreCompiler);
 
     if (task === 'info') {
-      await telemetryAction(
-        sys,
-        { flags: createConfigFlags({ task: 'info' }), logger, outputTargets: [] },
-        coreCompiler,
-        async () => {
-          await taskInfo(coreCompiler, sys, logger);
-        }
-      );
+      await taskInfo(coreCompiler, sys, logger);
```

Comment:
I think we can drop the `await` while we're in here too

Reply:
good eye! https://github.com/ionic-team/stencil/pull/3487/commits/d48eff4a5b77b9c6fb3bbb6f782e5a986ea8b6b8